jbt-it/neomdb
# #347 client workshop instance feedback admin

How to test:

- go to workshops -> Projektmanagement


files to check:
- WorkshopInstanceEvaluation
- WorkshopInstanceEvaluationAccordions
- WorkshopInstanceTextEvaluation
- WorkshopInstanceEvaluationTable
